### PR TITLE
Various improvements and suggestions

### PR DESCRIPTION
**Pull request by jonathantneal**
Changes:

**Props**:
- All [global HTML & SVG props](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/Props.ts) added to Props. <sup>[1](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L12) [2](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L56)</sup>

**Files**:
- All compiled files written to `dist`. <sup>[1](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L47)</sup> <sup>[2](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L69-L73)</sup>
- Feather MIT license added to all compiled files and README. <sup>[1](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L7)</sup> <sup>[2](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/README.md?plain=1#L84-L91)</sup>
- Brand names respectfully cased. <sup>[1](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/build.js#L63)</sup>
- 'Icon' suffix stripped from export. <sup>[1](https://github.com/jonathantneal/astro-feather-icons/blob/jn.update-project/packages/astro-feather-icons/README.md?plain=1#L19-L43)</sup>

**Dependencies**:
- All dependencies removed.
- Built-in Node API compiles files.
- Astro should not be a dependency, because it is not directly referenced.

**README.md**:
- Header updated to title case, with description from the original project.
- Install instructions updated to include yarn.

[Preview of README.md](https://github.com/jonathantneal/astro-feather-icons/tree/jn.update-project/packages/astro-feather-icons)

### CODE REVIEW

File: packages/astro-feather-icons/README.md [hunk 1]
```diff
@@ -1,45 +1,81 @@
...
+
+
 ## Usage
 
+All of the icons are available from a single import, and only the icons that are used will be added to the page.
+
 ```astro
 ---
-import { AirplayIcon, AtSignIcon, ... } from 'astro-feather-icons'
+import * as Icon from 'astro-feature-icons'
```
Comment (natemoo-re) [inline packages/astro-feather-icons/README.md]:
```suggestion
import * as Icon from 'astro-feather-icons'
```

File: packages/astro-feather-icons/README.md [hunk 2]
```diff
@@ -1,45 +1,81 @@
...
 
-<AirplayIcon size="24" />
-<AtSignIcon size="1.5x" />
+The icons are also available as individual imports.
+
+```astro
+---
+import AirPlayIcon from 'astro-feature-icons/AirPlay'
+import AtSignIcon from 'astro-feature-icons/AtSign'
+import ZapIcon from 'astro-feature-icons/Zap'
```
Comment (natemoo-re) [inline packages/astro-feather-icons/README.md]:
```suggestion
import AirPlayIcon from 'astro-feather-icons/AirPlay'
import AtSignIcon from 'astro-feather-icons/AtSign'
import ZapIcon from 'astro-feather-icons/Zap'
```

File: packages/astro-feather-icons/README.md (hunk 1 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md]: Thank you! Fixed in https://github.com/edazpotato/astro-feather-icons/pull/1/commits/32d4d5920ec4625a79b1702021215c301f03d060

File: packages/astro-feather-icons/README.md (hunk 2 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md]: Thank you! Fixed in https://github.com/edazpotato/astro-feather-icons/pull/1/commits/32d4d5920ec4625a79b1702021215c301f03d060

File: packages/astro-feather-icons/README.md [hunk 3]
```diff
@@ -1,45 +1,91 @@
...
+
+
+
 ## Usage
 
+All of the icons are available from a single import, and only the icons that are used will be added to the page.
+
 ```astro
 ---
-import { AirplayIcon, AtSignIcon, ... } from 'astro-feather-icons'
```
Comment (edazpotato) [inline packages/astro-feather-icons/README.md:17]: Is this style of import still possible with your changes?

File: packages/astro-feather-icons/README.md [hunk 4]
```diff
@@ -1,45 +1,91 @@
...
+import * as Icon from 'astro-feather-icons'
 ---
+<Icon.AirPlay />
+<Icon.AtSign />
+<Icon.Zap size="60" fill="gold" />
+```
+
+The icons are also available as individual imports.
 
-<AirplayIcon size="24" />
```
Comment (edazpotato) [inline packages/astro-feather-icons/README.md:20]: Can we please keep this example in the `README.md`, as this is the way I like using Icons.

**Review by edazpotato — CHANGES_REQUESTED** [review]
This looks really awsome! I just have a couple of changes I'd like to request before I merge this:
1. Deleting the package-lock.json file breaks the GitHub action (it uses `npm ci` rather than `npm i` for speed) so if you could undo that change I'd appreciate it.
2. I've added a couple of notes to the `README.md` file.

File: packages/astro-feather-icons/README.md (hunk 3 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md:17]: Yes, this change still allows individual named imports.

Note: The names would not include the 'Icon' suffix by default:

```js
import { AirPlay } from 'astro-feather-icons';
```

Therefore, to rename them:

```js
import { AirPlay as AirPlayIcon } from 'astro-feather-icons';
```

Another note: To have it _both_ ways would require duping the exports.

File: packages/astro-feather-icons/README.md (hunk 4 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md:20]: Yes, of course.

File: packages/astro-feather-icons/README.md (hunk 3 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md:17]: Done in 32d4d5920ec4625a79b1702021215c301f03d060

File: packages/astro-feather-icons/README.md (hunk 4 above)
Comment (jonathantneal) [reply packages/astro-feather-icons/README.md:20]: Done in 32d4d5920ec4625a79b1702021215c301f03d060

**Comment by jonathantneal** [comment]
Hey @edazpotato, thanks for the feedback. Please review the changes to be sure they’re as you want it.

**Review by edazpotato — APPROVED** [review]
Looks good to me!
Thanks so much for making this pull request. This repo was a fork of a fork so it did really need some cleaning up of the code.

**Pull request merged**